pkg/proxy/redirect.go

```diff
@@ -52,6 +52,7 @@ type Redirect struct {
 	// ProxyPort is the port the redirects redirects to where the proxy is
 	// listening on
 	ProxyPort      uint16
+	dstPort        uint16
```

Comment:
can you add documentation WRT `dstPort`?

Reply:
Done for the next update (its the L4 port the redirect is for, e.g., 80).